.github/workflows/CI-CHGLOG.yml

```
@@ -0,0 +1,369 @@
+---
+name: CI-CHGLOG
+description: "Continuous Integration workflow for Generating the CHANGELOG.md file."
```

actionlint (1.7.7)

unexpected key "description" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)